src/eval_framework/llm/vllm.py

```diff
@@ -460,14 +462,23 @@ def __init__(
 
         # Remove download_path from kwargs
         download_path = kwargs.pop("download_path", None)
-        with self.download_wandb_artifact(
-            artifact_name, version, user_supplied_download_path=download_path
-        ) as local_artifact_path:
+
+        with WandbFs(user_supplied_download_path=download_path) as wandb_fs:
```

Comment:
Might be a nit-pick, but I would vote for renaming `user_supplied_download_path` to `download_path`